src/sections/stats/sections/POL/components/ChartsWrapper/ChartsWrapper.tsx

```diff
@@ -35,7 +35,7 @@ export const ChartsWrapper = ({ assetSymbol }: Props) => {
       >
         <div sx={{ flex: "row", gap: [4, 12], justify: ["end", "start"] }}>
           <SChartTab
-            disabled={!isApi}
+            disabled={!isApi || true}
```

Comment:
Is this one correct? Always disabled?

Reply:
yes, since we don't have data for the pol chart